oktapam/client/cloud_connection_test.go

```diff
@@ -0,0 +1,185 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/okta/terraform-provider-oktapam/oktapam/utils"
+)
+
+func TestValidateCloudConnectionName(t *testing.T) {
+	data1 := CloudConnection{
+		Name:     utils.AsStringPtr("test_test"),
+		Provider: utils.AsStringPtr("aws"),
+		CloudConnectionDetails: &CloudConnectionDetails{
+			AccountId:  utils.AsStringPtr("123456789012"),
+			ExternalId: utils.AsStringPtr("external_id_value"),
```

Comment:
do we want to put validation for `externalId` being a uuid within the `validateCloudConnectionData` method?

Reply:
Done.